internal/runtime/client/client.go

```diff
@@ -502,32 +504,36 @@ func urlForExtension(config runtimev1.ClientConfig, gvh runtimecatalog.GroupVers
 		}
 	} else {
 		if config.URL == nil {
-			return nil, errors.New("at least one of Service and URL should be defined in config")
+			return nil, errors.New("failed to compute URL: at least one of service and url should be defined in config")
```

Comment:
Question (also in webhook comments) Shouldn't we have url capitalized? Completely non blocking and I don't care if it's inconsistent, just wondering about future errors / comments - my understanding is that capitalized acronyms should always be capitalized.

Reply:
I captialized the way it is in the CRD and the CRD uses the JSON capitalization.